### ISSUE TITLE
test failures with pypi distfile

### ISSUE THREAD

**Issue by 0-wiz-0** [miyakogi/m2r#9]
The pypi distfile of 0.1.9 has a number of test failures for me on NetBSD/python-3.6.2:
````
======================================================================
ERROR: test_dryrun (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 63, in test_dryrun
    with open(target_file) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.rst'

======================================================================
ERROR: test_dryrun (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 39, in tearDown
    f.write(self._orig_rst)
AttributeError: 'TestConvert' object has no attribute '_orig_rst'

======================================================================
ERROR: test_overwrite_file (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 89, in test_overwrite_file
    main()
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 572, in main
    output = parse_from_file(file)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 547, in parse_from_file
    raise OSError('No such file exists: {}'.format(file))
OSError: No such file exists: /scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.md

======================================================================
ERROR: test_overwrite_option (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 105, in test_overwrite_option
    main()
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 572, in main
    output = parse_from_file(file)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 547, in parse_from_file
    raise OSError('No such file exists: {}'.format(file))
OSError: No such file exists: /scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.md

======================================================================
ERROR: test_parse_file (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 55, in test_parse_file
    output = parse_from_file(test_md)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 547, in parse_from_file
    raise OSError('No such file exists: {}'.format(file))
OSError: No such file exists: /scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.md

======================================================================
ERROR: test_underscore_option (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 117, in test_underscore_option
    main()
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 572, in main
    output = parse_from_file(file)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 547, in parse_from_file
    raise OSError('No such file exists: {}'.format(file))
OSError: No such file exists: /scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.md

======================================================================
ERROR: test_write_file (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 77, in test_write_file
    main()
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 572, in main
    output = parse_from_file(file)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 547, in parse_from_file
    raise OSError('No such file exists: {}'.format(file))
OSError: No such file exists: /scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.md

======================================================================
ERROR: test_dryrun (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 68, in test_dryrun
    main()
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 572, in main
    output = parse_from_file(file)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 547, in parse_from_file
    raise OSError('No such file exists: {}'.format(file))
OSError: No such file exists: /scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.md

======================================================================
ERROR: test_overwrite_file (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 89, in test_overwrite_file
    main()
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 572, in main
    output = parse_from_file(file)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 547, in parse_from_file
    raise OSError('No such file exists: {}'.format(file))
OSError: No such file exists: /scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.md

======================================================================
ERROR: test_overwrite_option (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 105, in test_overwrite_option
    main()
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 572, in main
    output = parse_from_file(file)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 547, in parse_from_file
    raise OSError('No such file exists: {}'.format(file))
OSError: No such file exists: /scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.md

======================================================================
ERROR: test_parse_file (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 55, in test_parse_file
    output = parse_from_file(test_md)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 547, in parse_from_file
    raise OSError('No such file exists: {}'.format(file))
OSError: No such file exists: /scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.md

======================================================================
ERROR: test_underscore_option (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 117, in test_underscore_option
    main()
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 572, in main
    output = parse_from_file(file)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 547, in parse_from_file
    raise OSError('No such file exists: {}'.format(file))
OSError: No such file exists: /scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.md

======================================================================
ERROR: test_write_file (tests.test_cli.TestConvert)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_cli.py", line 77, in test_write_file
    main()
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 572, in main
    output = parse_from_file(file)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/m2r.py", line 547, in parse_from_file
    raise OSError('No such file exists: {}'.format(file))
OSError: No such file exists: /scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test.md

======================================================================
FAIL: test_python_code_block (tests.test_renderer.TestCodeBlock)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 303, in test_python_code_block
    out = self.conv(src)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 16, in conv
    self.check_rst(out)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 34, in check_rst
    self.assertLess(pub.document.reporter.max_level, 0)
AssertionError: 2 not less than 0

======================================================================
FAIL: test_python_code_block_indent (tests.test_renderer.TestCodeBlock)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 313, in test_python_code_block_indent
    out = self.conv(src)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 16, in conv
    self.check_rst(out)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 34, in check_rst
    self.assertLess(pub.document.reporter.max_level, 0)
AssertionError: 2 not less than 0

======================================================================
FAIL: test_code (tests.test_renderer.TestConplexText)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 590, in test_code
    self.conv(src)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 16, in conv
    self.check_rst(out)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 34, in check_rst
    self.assertLess(pub.document.reporter.max_level, 0)
AssertionError: 2 not less than 0

======================================================================
FAIL: test_python_code_block (tests.test_renderer.TestCodeBlock)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 303, in test_python_code_block
    out = self.conv(src)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 16, in conv
    self.check_rst(out)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 34, in check_rst
    self.assertLess(pub.document.reporter.max_level, 0)
AssertionError: 2 not less than 0

======================================================================
FAIL: test_python_code_block_indent (tests.test_renderer.TestCodeBlock)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 313, in test_python_code_block_indent
    out = self.conv(src)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 16, in conv
    self.check_rst(out)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 34, in check_rst
    self.assertLess(pub.document.reporter.max_level, 0)
AssertionError: 2 not less than 0

======================================================================
FAIL: test_code (tests.test_renderer.TestConplexText)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 590, in test_code
    self.conv(src)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 16, in conv
    self.check_rst(out)
  File "/scratch/textproc/py-m2r/work/m2r-0.1.9/tests/test_renderer.py", line 34, in check_rst
    self.assertLess(pub.document.reporter.max_level, 0)
AssertionError: 2 not less than 0

----------------------------------------------------------------------
Ran 148 tests in 1.371s

FAILED (failures=6, errors=13, skipped=2)
````

Some of them seem to be because `test.md` is not included in the tarball.
I'm not sure about the other ones.

**Comment by miyakogi** [comment]
Thank you for reporting.
This problem would be fixed in v0.1.10

**Comment by 0-wiz-0** [comment]
Thank you, all tests work fine with 0.1.10:
````
Ran 74 tests in 0.789s

OK (skipped=1)
````
